nautobot/utilities/forms/forms.py

```diff
@@ -21,7 +21,12 @@
 
 
 class AddressFieldMixin(forms.ModelForm):
-    """ ModelForm mixin for IPAddress based models """
+    """
+    ModelForm mixin for IPAddress based models.
+
+    IMPORTANT: because this mixin's save() method discards any arguments such as commit=True,
+    it MUST be last in any inheritance sequence.
+    """
```

Comment:
Is this still accurate since this is no longer overloading `save()`

Reply:
Nope, no longer accurate - thanks!